.github/workflows/release.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         upload_url: https://uploads.github.com/repos/FreeTubeApp/FreeTube/releases/${{ secrets.UPLOAD_ID }}/assets{?name,label}
-        asset_name: freetube-${{ steps.getPackageInfo.outputs.version }}-linux-portable-x64.zip
+        asset_name: freetube-${{ steps.getPackageInfo.outputs.version }}-linux-x64-portable.zip
```

Comment:
I think the flatpak.yml file would need to be updated in a few places.

Ex:
https://github.com/FreeTubeApp/FreeTube/blob/550ce7739bbee8662042cae148cd8e84ebb5542d/.github/workflows/flatpak.yml#L54